client/python/conductor/conductor.py

```diff
@@ -267,6 +283,41 @@ def restartWorkflow(self, wfId, taskRefName, fromTaskRef):
         params['from'] = fromTaskRef
         self.post(url, params, None)
 
+class EventServicesClient(BaseClient):
+    BASE_RESOURCE = 'event'
+
+    def __init__(self, baseURL):
+        BaseClient.__init__(self, baseURL, self.BASE_RESOURCE)
+
+    def getEventHandlerDef(self, event, activeOnly=True):
+        url = self.makeUrl('{}', event)
+        params = {}
+        params['activeOnly'] = activeOnly
+        return self.get(url, params)
+
+    def getEventHandlerDefs(self):
+        url = self.makeUrl('')
```

Comment:
Minor: you could give 1st argument a default value so you don't have to pass an empty format, like so:
```
    def makeUrl(self, urlformat=None, *argv):
        url = self.baseResource + '/'
        if urlformat:
             url += urlformat.format(*argv)
        return url
```